imagetagger/imagetagger/annotations/static/annotations/js/annotations.js

```diff
@@ -258,6 +258,9 @@ function calculateImageScale() {
         globals.currentAnnotations = globals.allAnnotations.filter(function(e) {
           return e.annotation_type.id === gAnnotationType;
         });
+        let annotatedPic = document.getElementById("annotate_image_link_" + gImageId);
```

Comment:
It looks like the surrounding code uses jQuery; can we do that here too?

Something like 

```js
$("#annotate_image_link_" + gImageId).addClass("annotated");
```

It's the same thing (if I did it right) but using jQuery rather than DOM functions, just to match the surrounding style.